core/dbt/task/run_operation.py

```diff
@@ -41,7 +41,7 @@ def _run_unsafe(self, package_name, macro_name) -> agate.Table:
         with adapter.connection_named("macro_{}".format(macro_name)):
             adapter.clear_transaction()
             res = adapter.execute_macro(
-                macro_name, project=package_name, kwargs=macro_kwargs, manifest=self.manifest
+                macro_name, project=package_name, kwargs=macro_kwargs, macro_resolver=self.manifest
```

Comment:
it may be possible to remove the macro_resolver usage entirely but I'd like to do that in a follow-on to keep this PR more self-contained.